app/controllers/reviews_controller.rb

```diff
@@ -6,8 +6,13 @@ def new
 
   def create
     @shelter = Shelter.find(params[:id])
-    @shelter.reviews.create(review_params)
-    redirect_to "/shelters/#{@shelter.id}"
+    review = @shelter.reviews.new(review_params)
```

Comment:
I'm not entirely sure of the difference between .new and .create, I will go through the lesson and educate myself.